src/document_formatting_edit_provider/dbtDocumentFormattingEditProvider.ts

```diff
@@ -69,27 +67,10 @@ export class DbtDocumentFormattingEditProvider
           .createCommandProcessExecution({
             command: sqlFmtPath,
             args: sqlFmtArgs,
-            stdin: document.getText(),
           })
           .complete();
         return [];
-      } catch (diffOutput) {
-        try {
-          return this.processDiffOutput(document, diffOutput as string);
-        } catch (error) {
-          this.telemetry.sendTelemetryError(
-            "formatDbtModelApplyDiffFailed",
-            error,
-          );
-          window.showErrorMessage(
-            extendErrorWithSupportLinks(
-              "Could not process difference output from sqlfmt. Detailed error: " +
-                error +
-                ".",
-            ),
-          );
-        }
-      }
+      } catch (sqlfmtOutput) {}
```

Comment:
How do we verify success here? We are ignoring the output?

Reply:
Indeed, the stdout is always something like "1 file changed, 0 file left unchanged". Parsing the "expected successful message" doesn't sound like a great idea because I could imagine that they might feel more free to change it than the diff or some error code. I guess we could simply use the return error code 0 from the shell but I wasn't sure how to do that because the try/catch looks to only capture the stdout/stderr. I can try to have a look this weekend to understand how to get the shell output code and use that for validation and keep the telemetry call.

Reply:
Fixed! Let me know what you think. I don't understand why it looks like sqlfmt systematically write to stderr, but anyway the shell error code looks more reliable to detect errors.